windows/power_operation_scripts/set_power_script.yml

```diff
@@ -0,0 +1,152 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
```

Comment:
It seems we did not make this very clear. The "Description, Parameters, Return" parts are needed in task files in "common" or "utils" folders, since these tasks will be used commonly by others. Here task files in test cases folder, only comments are ok.